examples/klotho-analysis/klotho-go-example/README.md

```diff
@@ -0,0 +1,69 @@
+# Simple Klotho app
+
+## Install Klotho
+
+### In linux
+
+Run
+
+```bash
+curl -fsSL "http://srv.klo.dev/update/latest/linux/x86_64?stream=pro:latest" -o klotho
```

Comment:
It´d be good to put the link to the official documentation where this way of installing is suggested (notice HTTP is not secured). it´d be worth trying out the docker image (https://hub.docker.com/r/klothoplatform/klotho)